### PR TITLE
 v1.3 config refactor and new options added

### PR DESCRIPTION
**Pull request by chuan-wang**
I will add the function for reporting software versions in the next PR.

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -10,17 +10,18 @@ vim: syntax=groovy
...
 profiles {
+
   standard {
     includeConfig 'conf/base.config'
     includeConfig 'conf/uppmax.config'
     includeConfig 'conf/igenomes.config'
   }
   devel {
-    includeConfig 'conf/base.config'
-    includeConfig 'conf/uppmax.config'
```
Comment (ewels) [inline nextflow.config]: I think you need these other config includes still.. (also for aws)

**Review by ewels — COMMENTED** [review]
Looks brilliant!

File: nextflow.config (hunk 1 above)
Comment (chuan-wang) [reply nextflow.config]: Thanks Phil! They have been added.

File: nextflow.config [hunk 2]
```diff
@@ -10,15 +10,18 @@ vim: syntax=groovy
...
 profiles {
+
   standard {
     includeConfig 'conf/base.config'
     includeConfig 'conf/uppmax.config'
     includeConfig 'conf/igenomes.config'
   }
   devel {
     includeConfig 'conf/base.config'
-    includeConfig 'conf/uppmax.config'
```
Comment (ewels) [inline nextflow.config:21]: Nearly! You need the main uppmax one still for the uppmax devel.

**Comment by ewels** [comment]
Now we're suddenly getting an error on Travis, not sure why?

```
Error in library(spp) : there is no package called ‘spp’
```

Any ideas?

**Comment by chuan-wang** [comment]
Looks like a version issue. I will look into it. /chuan

**Comment by chuan-wang** [comment]
Bug fixed. Tested locally and worked as expected.

**Comment by ewels** [comment]
👍

**Pull request merged**